Add small note on revocation statement in GCP CloudSQL documentation (#23301)
hashicorp/vault: website/content/docs/secrets/databases/mysql-maria.mdx

        service_account_json="@my_credentials.json"
    ```

> 1.  Configure a role in Vault as done above. Here you can override Vault's default revocation statements to simply
>     drop the user:

Reviewer: ```suggestion
1. Configure a new role in Vault but override the default revocation statements
   so Vault will drop the user instead:
```
Style correction: write in declarative statements

Author: Updated, thanks!

    ```shell-session
       $ vault write database/roles/my-role \
           db_name=my-mysql-database \
           creation_statements="CREATE USER '{{name}}'@'%' IDENTIFIED BY '{{password}}';GRANT SELECT ON *.* TO '{{name}}'@'%';" \
           revocation_statements="DROP USER '{{name}}'@'%';" \
           default_ttl="1h" \
           max_ttl="24h"
    ```

> Once the role configuration is complete, the steps generate credentials are the same as the ones listed above.

Reviewer 2: ```suggestion
Once the role configuration is complete, the steps to generate credentials are the same as the ones listed above.
```

> Once the role configuration is complete, the steps generate credentials are the same as the ones listed above.

Reviewer: ```suggestion
When you finish configuring the new role, generate credentials as before:
```

Style correction: write in declarative statements for instructions.

> Once the role configuration is complete, the steps generate credentials are the same as the ones listed above.

Reviewer: We should list the actual commands here, not send the reader elsewhere in the doc. If we expect to use the steps under multiple situations, we can always make those steps a partial so it's easier to repeat them.
